### ISSUE TITLE
Questions about the dvo code

### ISSUE THREAD

**Issue by Ethan-Zhou** [tum-vision/dvo#9]
Hi, 
Recently, I began to study something about dense image registration. And I am new to C++. 
I just want to use your codes to do some static dense image registration work. Thus, I would like to ask whether it is enough to download the dvo_core part of the this lab?
Thanks

**Comment by christiankerl** [comment]
Yes dvo_core contains everything related to dense image alignment.

Best, Christian

**Comment by Ethan-Zhou** [comment]
Thanks for your reply, Mr Christian.

I just have download the dvo_core and successfully compile it.

I have read your thesis and got your brilliant ideas. But I am not very familiar with C++. Thus, if possible, could you show me a demo (e.g. main.cpp) explaining how to estimate the camera motion using a pair of adjacent frames . Just several rows of codes will be very helpful. 

My mailbox is 
                                cavatina@yeah.net

Thanks in advance

**Comment by Ethan-Zhou** [comment]
Hi Christian,
There is a class called QMatrix in the Transform.h file. I cannot find where it was defined. Could you point it out for me. Thanks.

**Comment by christiankerl** [comment]
which file do you refer to?

**Comment by Ethan-Zhou** [comment]
Thanks for your reply. I just found the Transform.h file belongs to Eigen, so the QMatrix may be a class of another 3rd party lib.
